src/isql/isql.epp

```diff
@@ -244,6 +244,7 @@ namespace IcuUtil
 	}
 }
 
+void ISQL_warning(Firebird::CheckStatusWrapper* st);
```

Comment:
It would be better to fix declaration in isql_proto.h.

Reply:
Indeed. Didn't see it there.

Reply:
@dmitry-starodubov Will you apply changes, suggested by DS, or I can do it myself?

Reply:
Done